stable/shell/values.yaml

```
@@ -58,6 +58,15 @@ daemon:
     tag: "1.0.0"
     pullPolicy: "IfNotPresent"
 
+sshServer:
+  enabled: false
+
+  # can specifically set the port for the service NodePort (optional)
+  # port:
```

```suggestion
  # nodePort:
```
Also please add to the comment that if one is not given, a random one will be assigned